lib/rspec/matchers/power_assert_matchers.rb

```diff
@@ -26,7 +26,7 @@ module PowerAssertMatchers
   match(notify_expectation_failures: true) do |actual|
     case actual
     when Proc
-      result = ::PowerAssert.start(actual, assertion_method: __callee__) do |pa|
+      result = PowerAssert.start(actual, assertion_method: __callee__) do |pa|
```

Comment:
Afaik, ruby does not change name resolutions even if evaluated the given block in eval families. So I think this change is acceptable 🤔